Add keyboard 3dp660 (#10296)
qmk/qmk_firmware: keyboards/handwired/3dp660/readme.md

# 3dp660

> <img src="https://imgur.com/KWyU57e" alt="Clueboard660" width="400"/>

Reviewer: ```suggestion

![](https://i.imgur.com/KWyU57el.jpg)
```

# 3dp660

![](https://i.imgur.com/KWyU57el.jpg)

### Teensy 2.0 Pinout

> | rows        | 0   | 1   | 2   | 3   | 4   |
> | ----------- | --- | --- | --- | --- | --- |
> | Teensy pin  | B0  | B1  | B2  | B3  | B7  |
> | qmk pin     | B0  | B1  | B2  | B3  | B7  |
> 
> | columns     | 0   | 1   | 2   | 3   | 4   | 5   | 6   | 7   | 8   | 9   | 10  | 11  | 12  | 13  | 14  |
> | ----------- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
> | Teensy pin  | D2  | D3  | C6  | C7  | D5  | D4  | D7  | B4  | B5  | B6  | F7  | F6  | F5  | F4  | F1  |
> | qmk pin     | D2  | D3  | C6  | C7  | D5  | D4  | D7  | B4  | B5  | B6  | F7  | F6  | F5  | F4  | F1  |

Reviewer 2: You could do:

```suggestion
| rows            | 0   | 1   | 2   | 3   | 4   |
| --------------- | --- | --- | --- | --- | --- |
| Teensy/QMK pin  | B0  | B1  | B2  | B3  | B7  |

| columns         | 0   | 1   | 2   | 3   | 4   | 5   | 6   | 7   | 8   | 9   | 10  | 11  | 12  | 13  | 14  |
| --------------- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Teensy/QMK pin  | D2  | D3  | C6  | C7  | D5  | D4  | D7  | B4  | B5  | B6  | F7  | F6  | F5  | F4  | F1  |
```

since the pin names are the same between Teensy and QMK. But I consider this optional.

Make example for this keyboard (after setting up your build environment):

    make handwired/3dp660:default

> Bootmagic is enabled.  Press the key at (0,0) while plugging the keyboard in to jump to bootloader. This is numlock in the default keymap.

Reviewer 2: Goes with the suggestion of changing `BOOTMAGIC_ENABLE` to `lite`. Allows the user to reset to bootloader mode by holding a key while connecting the USB cable. Potentially more accessible than hitting the reset key on the Teensy.

```suggestion
**Reset Key:** Hold down the key located at `k00`, commonly programmed as Escape while plugging in the keyboard.
```

> * Keyboard Maintainer: [gooberpsycho]()

Reviewer: ```suggestion
* Keyboard Maintainer: [gooberpsycho](https://github.com/gooberpsycho)
```
